io.trustable.stpadocumentationtool.json

```diff
@@ -0,0 +1,73 @@
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "stpa_documentation_tool",
+    "rename-desktop-file": "stpa_documentation_tool.desktop",
+    "rename-appdata-file": "stpa_documentation_tool.appdata.xml",
+    "finish-args": [ "--socket=x11",
+                     "--filesystem=xdg-download" ],
+    "modules": [
+        {
+            "name": "yaml-cpp",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.2.tar.gz",
+                    "sha256": "e4d8560e163c3d875fd5d9e5542b5fd5bec810febdcba61481fe5fc4e6b1fd05"
+                }
+            ],
+            "cleanup": [ "/include", "/libpkgconfig", "/lib/cmake", "/lib/*.a"]
+        },
+        {
+            "name": "stpadocumentationtool",
+            "buildsystem": "cmake-ninja",
+                "config-opts": [
+                    "-DBUILD=NO_UNIT_TESTS"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "tag": "v1.0",
+                    "commit": "b5364627b93f9eadfe04d7a3a12ac55488717c56",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool.git"
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool/raw/v1.0/flathub/share/icons/hicolor/128x128/stpa_documentation_tool-128x128.png",
+                    "sha256": "da9d4496b251b25369d5002b10d492b9c0abf2ea0b04ea49c975778e54138526"
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool/raw/v1.0/flathub/share/icons/hicolor/64x64/stpa_documentation_tool-64x64.png",
+                    "sha256": "50ab423c08f5412bd734562ebcebe517cd964ef63ba5e311803c8f42ce6ed671"
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool/raw/v1.0/flathub/share/icons/hicolor/48x48/stpa_documentation_tool-48x48.png",
+                    "sha256": "ab80a2dcf65b16123ed0bf81a5453522849292a9d8600249452b53c6a567e65c"
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool/raw/v1.0/flathub/share/stpa_documentation_tool.appdata.xml",
+                    "sha256": "4be3a28e8aa0bb786111ab6c68ac579e699f8809ca7551550300d8834e600f15"
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/trustable/stpa-documentation-tool/raw/v1.0/flathub/share/stpa_documentation_tool.desktop",
+                    "sha256": "71302852a090e83b486a96569471e4a90514003e140b093fc6312a2a17431366"
+                }
+
+            ],
+            "build-commands": [
```

Comment:
Ideally this should be above `sources` but it's a nitpick and can be fixed later.

Reply:
Got it - likewise, I did try to move the build commands into the cmake, but found enormous difficulty in getting the flatpak build to see the meta files - I decided a compromise was to leave them in the manfest but upstream them.